manifests/profile/named_instances/puma_wrapper.pp

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Wrapper for puma
+# It prefers system puma over bundled puma
```

Comment:
This comment is a little misleading. In the script, a bundled puma will be used if present, and fall back to the system puma -- and we know that system breaks if there is a different version bundled...

So I guess _we_ prefer system, but defer to any bundled version... right?